src/nvim/api/private/helpers.c

```diff
@@ -488,6 +488,15 @@ String cstr_to_string(const char *str)
   };
 }
 
+/// Copies a String to an allocated, NUL-terminated C string.
+///
+/// @param string the String to copy
+/// @return the resulting C string
+char *string_to_cstr(String string)
```

Comment:
nit: looks like `str` is more common in this file
```suggestion
char *string_to_cstr(String str)
```